mac/help/about/whatsnew.md

```diff
@@ -2,7 +2,7 @@
 title: What's New
 ---
 
-Here are some of the new features we have added to Keyman 15.0 for macOS:
+Here are some of the new features we have added to Keyman 16.0 for macOS:
 
 * Localizable UI through [translate.keyman.com](https://translate.keyman.com)
```

Comment:
Similar thing here - "what's new" sections shouldn't have "added to Keyman xx.0" version-shifted.